packages/ui/src/components/Designer/Sidebar/DetailView/index.tsx

```diff
@@ -58,16 +58,23 @@ const DetailView = (
   }, [activeSchema, activeElements, pluginsRegistry, JSON.stringify(options)]);
 
   useEffect(() => {
-    form.resetFields();
-    form.setValues({ ...activeSchema });
+    const values: any = { ...activeSchema };
```

Comment:
I know this code isn't very elegant. However, it was necessary to change the way position.x and position.y data are handled in order to create a three-column layout in form-render.